docs/src/reference/compilation.rst

```diff
@@ -134,17 +135,12 @@ in one line)::
     setup(
         name = "My hello app",
         ext_modules = cythonize(extensions),
+        zip_safe=False,
     )
 
-Note that when using setuptools, you should import it before Cython as
-setuptools may replace the ``Extension`` class in distutils.  Otherwise,
-both might disagree about the class to use here.
```

Comment:
This still seems worth mentioning, although some care has been taken now to work around this problem.